### DISCUSSION TITLE
Add user to group - filter by custom field does not filter (v 1.11.26)

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5491] [General]
Hello, I have a problem here `/main/admin/add_users_to_usergroup.php?id=`
I'm trying to filter all users by a couple of custom fields, but when I apply the filter nothing happens. I would like the same behaviour of the form that allows to insert users into courses, in that page the filter works. Is there something more to configure or is it a bug?

**Comment by LudiscapeXApi** [comment]
It seems that custom fields are not working properly with the filters.

The development team appears to be currently focused on Chamilo version 2. So, if you wish to improve this part, either you have a developer who can contribute to fixing the issue, or you can reach out to an official provider here: https://chamilo.org/en/providers/.

You could also consider workarounds such as using official code with a specific identifier for the context.
